slices/string_test.go

```diff
@@ -7,23 +7,23 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-func Test_String_Scan(t *testing.T) {
-	r := require.New(t)
-	in := `{"This has a comma,","This has a double quote\"","Also a single'"}`
-	s := &String{}
-	r.NoError(s.Scan(in))
-	ss := []string(*s)
-	r.Len(ss, 3)
-	r.Equal([]string{"This has a comma,", "This has a double quote\"", "Also a single'"}, ss)
-}
-
-func Test_String_Value(t *testing.T) {
-	r := require.New(t)
-	s := String{"This has a comma,", "This has a double quote\"", "Also a single'"}
-	v, err := s.Value()
-	r.NoError(err)
-	r.Equal(`{"This has a comma,","This has a double quote\"","Also a single'"}`, v)
-}
+// func Test_String_Scan(t *testing.T) {
+// 	r := require.New(t)
+// 	in := `{"This has a comma,","This has a double quote\"","Also a single'"}`
+// 	s := &String{}
+// 	r.NoError(s.Scan(in))
+// 	ss := []string(*s)
+// 	r.Len(ss, 3)
+// 	r.Equal([]string{"This has a comma,", "This has a double quote\"", "Also a single'"}, ss)
+// }
+//
+// func Test_String_Value(t *testing.T) {
+// 	r := require.New(t)
+// 	s := String{"This has a comma,", "This has a double quote\"", "Also a single'"}
+// 	v, err := s.Value()
+// 	r.NoError(err)
+// 	r.Equal(`{"This has a comma,","This has a double quote\"","Also a single'"}`, v)
+// }
```

Comment:
Is it commented because it doesn't work, or because it's not required anymore? If it doesn't work, we need to fix them, if it's not relevant anymore, we can drop them. :)

Reply:
No, this should be reverted!